### PR TITLE
Move test/new/* into test/ ##tests

### PR DESCRIPTION
**Pull request by radare**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [x] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

**Detailed description**

Move test/new into test.. basically now we have test/db.. and i would suggest moving test/scripts into test/bins/src

**Test plan**

just wait for the ci

**Closing issues**

yup. but i cant find it now

### CODE REVIEW

**Comment by XVilka** [comment]
Can't open the bins:
```
[XX] db/cmd/cmd_zignature af ; af ; zfs libc-v7.sig
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 's 0x4e2420 ; af ; s 0x4e25c7 ; af ; zfs ../bins/other/sigs/libc-v7.sig ; afl ~4e2420' bins/elf/analysis/pid_stripped
-- stdout


-- stderr
Can't open ../bins/other/sigs/libc-v7.sig

[XX] db/cmd/cmd_zignature zfd libc-v7.sig
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'zfd ../bins/other/sigs/libc-v7.sig' bins/elf/analysis/pid_stripped
-- stdout


-- stderr
Can't open ../bins/other/sigs/libc-v7.sig

[XX] db/cmd/cmd_zignature af ; af ; zfs libc-v10.sig
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 's 0x4e2420 ; af ; s 0x4e25c7 ; af ; zfs ../bins/other/sigs/libc-v10.sig ; afl ~4e2420' bins/elf/analysis/pid_stripped
-- stdout


-- stderr
Can't open ../bins/other/sigs/libc-v10.sig

[XX] db/cmd/cmd_zignature zfd libc-v10.sig
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'zfd ../bins/other/sigs/libc-v10.sig' bins/elf/analysis/pid_stripped
-- stdout


-- stderr
Can't open ../bins/other/sigs/libc-v10.sig

[XX] db/cmd/cmd_zignature aa ; zfs libc-v7.sig
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'aa ; zfs ../bins/other/sigs/libc-v7.sig' bins/elf/analysis/pid_stripped
-- stdout


-- stderr
[ ] Analyze all flags starting with sym. and entry0 (aa)
[
[x] Analyze all flags starting with sym. and entry0 (aa)

Can't open ../bins/other/sigs/libc-v7.sig

[**]                                   db/cmd/feat_yank     2559 OK        75 BR       87 XX       10 FX
[**]                                    db/cmd/cmd_yank     2559 OK        75 BR       87 XX       10 FX
[**]                                     db/cmd/cmd_wox     2559 OK        75 BR       87 XX       10 FX
[**]                                     db/cmd/cmd_wop     2559 OK        75 BR       87 XX       10 FX
[**]                                     db/cmd/cmd_wfs     2559 OK        75 BR       87 XX       10 FX
[XX] db/cmd/cmd_to to arrays
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'to ../bins/headers/s3.h
t S1
tk S1
tk struct.S1
tk struct.S1.x
tk struct.S1.y
tk struct.S1.z
' -
-- stdout
--- .a	2020-04-09 22:55:02.430307000 +0000
+++ .b	2020-04-09 22:55:02.430337000 +0000
@@ -1,6 +0,0 @@
-pf [3]d[4]dd x y z
-struct
-x,y,z
-int32_t,0,3
-int32_t,12,4
-int32_t,28,0


-- stderr
tcc: error: file '../bins/headers/s3.h' not found
Cannot find 'S1' type

[XX] db/cmd/cmd_to to typedefs
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'to ../bins/headers/t1.h
tk~easy,humans,days,tokyo,dangerous
' -
-- stdout
--- .a	2020-04-09 22:55:02.444945000 +0000
+++ .b	2020-04-09 22:55:02.444976000 +0000
@@ -1,13 +0,0 @@
-dangerous=struct
-days=typedef
-easy_prey=typedef
-humans=typedef
-struct.dangerous=neo
-struct.dangerous.!size=64
-struct.dangerous.neo=uint64_t,0,0
-struct.dangerous.neo.meta=13
-tokyo=typedef
-typedef.days=struct dangerous
-typedef.easy_prey=uint32_t
-typedef.humans=uint32_t
-typedef.tokyo=struct dangerous


-- stderr
tcc: error: file '../bins/headers/t1.h' not found

[**]                                  db/cmd/cmd_visual     2563 OK        75 BR       89 XX       10 FX
[XX] db/cmd/cmd_to to perturbated
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'to ../bins/headers/s1.h; to ../bins/headers/s4.h; t S1' -
-- stdout
--- .a	2020-04-09 22:55:02.453379000 +0000
+++ .b	2020-04-09 22:55:02.453413000 +0000
@@ -1 +0,0 @@
-pf ddd x test z


-- stderr
tcc: error: file '../bins/headers/s1.h' not found
tcc: error: file '../bins/headers/s4.h' not found
Cannot find 'S1' type

[XX] db/cmd/cmd_to to
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'to ../bins/headers/s2.h; t S1' -
-- stdout
--- .a	2020-04-09 22:55:02.707379000 +0000
+++ .b	2020-04-09 22:55:02.707418000 +0000
@@ -1 +0,0 @@
-pf ddd x test z


-- stderr
tcc: error: file '../bins/headers/s2.h' not found
Cannot find 'S1' type

[XX] db/cmd/cmd_to to
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'to ../bins/headers/s1.h; to ../bins/headers/s2.h; t S1' -
-- stdout
--- .a	2020-04-09 22:55:02.712731000 +0000
+++ .b	2020-04-09 22:55:02.712759000 +0000
@@ -1 +0,0 @@
-pf ddd x test z


-- stderr
tcc: error: file '../bins/headers/s1.h' not found
tcc: error: file '../bins/headers/s2.h' not found
Cannot find 'S1' type

[XX] db/cmd/cmd_to to
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'to ../bins/headers/s1.h;t S1' -
-- stdout


-- stderr
tcc: error: file '../bins/headers/s1.h' not found
Cannot find 'S1' type

[XX] db/cmd/cmd_zignature aa ; zfs libc-v10.sig
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'aa ; zfs ../bins/other/sigs/libc-v10.sig' bins/elf/analysis/pid_stripped
-- stdout


-- stderr
[ ] Analyze all flags starting with sym. and entry0 (aa)
[
[x] Analyze all flags starting with sym. and entry0 (aa)

Can't open ../bins/other/sigs/libc-v10.sig

[**]                                      db/cmd/cmd_to     2567 OK        75 BR       94 XX       10 FX
[**]                                  db/cmd/cmd_tmpbuf     2567 OK        75 BR       94 XX       10 FX
[**]                                     db/cmd/cmd_tcc     2573 OK        75 BR       94 XX       10 FX
[**]                                    db/cmd/cmd_task     2612 OK        76 BR       94 XX       10 FX
[**]                                      db/cmd/cmd_tc     2612 OK        76 BR       94 XX       10 FX
[**]                                   db/cmd/cmd_shift     2612 OK        76 BR       94 XX       10 FX
[**]                               db/cmd/cmd_search_in     2612 OK        76 BR       94 XX       10 FX
[XX] db/cmd/cmd_search search by part of file dec/hex
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'wx 8048 @ 0x10
/F ../bins/elf/analysis/hello.raw 0x1c 0x2
' -
-- stdout
--- .a	2020-04-09 22:55:06.767179000 +0000
+++ .b	2020-04-09 22:55:06.767205000 +0000
@@ -1 +0,0 @@
-0x00000010 hit0_0 8048


-- stderr
Cannot open '../bins/elf/analysis/hello.raw'

[XX] db/cmd/cmd_search search by rest of file dec/dec
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'wx 0e @ 0x10
/F ../bins/elf/analysis/hello.raw 1 1
' -
-- stdout
--- .a	2020-04-09 22:55:07.042122000 +0000
+++ .b	2020-04-09 22:55:07.042146000 +0000
@@ -1 +0,0 @@
-0x00000010 hit0_0 0e


-- stderr
Cannot open '../bins/elf/analysis/hello.raw'

[XX] db/cmd/cmd_search search by rest of file
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'wx 210a @ 0x10
/F ../bins/elf/analysis/hello.raw 48
' -
-- stdout
--- .a	2020-04-09 22:55:07.055277000 +0000
+++ .b	2020-04-09 22:55:07.055305000 +0000
@@ -1 +0,0 @@
-0x00000010 hit0_0 210a


-- stderr
Cannot open '../bins/elf/analysis/hello.raw'

[XX] db/cmd/cmd_search search by whole file
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'wx 90 @ 0x10
/F ../bins/other/radiff2/radiff2_c_2
' malloc://64
-- stdout
--- .a	2020-04-09 22:55:07.079255000 +0000
+++ .b	2020-04-09 22:55:07.079283000 +0000
@@ -1 +0,0 @@
-0x00000010 hit0_0 90


-- stderr
Cannot open '../bins/other/radiff2/radiff2_c_2'

[**]                                  db/cmd/cmd_system     2666 OK        76 BR       98 XX       10 FX
[**]                                    db/cmd/cmd_seek     2692 OK        78 BR       98 XX       10 FX
[XX] db/cmd/cmd_search pm olf
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -n -Qc 'pm ../bins/src/olf.magic' bins/elf/ioli/crackme0x00
-- stdout


-- stderr
Invalid argument
0x00000000 [0 matches found]
failed r_magic_load ("../bins/src/olf.magic") no magic files loaded! could not find any magic files!

[**]                                     db/cmd/cmd_rop     2716 OK        78 BR       99 XX       10 FX
[**]                                 db/cmd/cmd_repeats     2716 OK        78 BR       99 XX       10 FX
[**]                                       db/cmd/cmd_r     2716 OK        78 BR       99 XX       10 FX
[**]                                db/cmd/cmd_question     2724 OK        78 BR       99 XX       10 FX
[XX] db/cmd/cmd_ps ps+ libc++ 64bit
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -n -m 0x00600dc8 -Qc 'on ../bins/elf/stdstring64-libc++.heap 0x0128b000 > /dev/null
f obj.empty_str 24 0x006010b8
f obj.long_str1 24 0x00601088
f obj.short_str1 24 0x00601058
f obj.long_str2 24 0x006010a0
f obj.short_str2 24 0x00601070
e asm.bits=64
ps+ @ obj.short_str1
ps+ @ obj.short_str2
ps+ @ obj.empty_str
ps+ @ obj.long_str1
ps+ @ obj.long_str2
' bins/elf/stdstring64-libc++.LOAD1
-- stdout
--- .a	2020-04-09 22:55:12.204251000 +0000
+++ .b	2020-04-09 22:55:12.204302000 +0000
@@ -1,5 +1,5 @@
a
abcdefghijklmnopqrstuv

-abcdefghijklmnopqrstuvw
-abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ
+\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff
+\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff


-- stderr
WARNING: using oba to load the syminfo from different mapaddress.
TODO: Must use the API instead of running commands to speedup loading times.
Cannot open file '../bins/elf/stdstring64-libc++.heap'

[XX] db/cmd/cmd_ps ps+ libc++ 32bit
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -n -m 0x08049ee8 -Qc 'on ../bins/elf/stdstring32-libc++.heap 0x09eb8000 > /dev/null
f obj.empty_str 12 0x0804a064
f obj.long_str1 12 0x0804a04c
f obj.short_str1 12 0x0804a034
f obj.long_str2 12 0x0804a058
f obj.short_str2 12 0x0804a040
e asm.bits=32
ps+ @ obj.short_str1
ps+ @ obj.short_str2
ps+ @ obj.empty_str
ps+ @ obj.long_str1
ps+ @ obj.long_str2
' bins/elf/stdstring32-libc++.LOAD1
-- stdout
--- .a	2020-04-09 22:55:12.212716000 +0000
+++ .b	2020-04-09 22:55:12.212762000 +0000
@@ -1,5 +1,5 @@
a
abcdefghij

-abcdefghijk
-abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ
+\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff
+\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff


-- stderr
WARNING: using oba to load the syminfo from different mapaddress.
TODO: Must use the API instead of running commands to speedup loading times.
Cannot open file '../bins/elf/stdstring32-libc++.heap'

[**]                                     db/cmd/cmd_pxd     2753 OK        82 BR      101 XX       10 FX
[**]                                     db/cmd/cmd_pxw     2753 OK        82 BR      101 XX       10 FX
[**]                                      db/cmd/cmd_px     2753 OK        82 BR      101 XX       10 FX
[**]                                     db/cmd/cmd_psj     2753 OK        82 BR      101 XX       10 FX
[**]                                      db/cmd/cmd_ps     2753 OK        82 BR      101 XX       10 FX
[XX] db/cmd/cmd_project Import mapped project
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'e dir.projects = ../bins/other/projects/
Po resetc64 > /dev/null
pu 3 @0xc000
' malloc://512
-- stdout


-- stderr
Cannot open project info (/usr/home/build/radare2/test/../bins/other/projects//resetc64/rc)
Cannot retrieve information for project '/usr/home/build/radare2/test/../bins/other/projects//resetc64/rc'

[XX] db/cmd/cmd_project Import project
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'e dir.projects = ../bins/other/projects
Po test > /dev/null
C*
' malloc://512
-- stdout
--- .a	2020-04-09 22:55:13.002428000 +0000
+++ .b	2020-04-09 22:55:13.002456000 +0000
@@ -1 +0,0 @@
-CCu base64:U29tZSB0ZXN0 @ 0x00404890


-- stderr
Cannot open project info (/usr/home/build/radare2/test/../bins/other/projects/test/rc)
Cannot retrieve information for project '/usr/home/build/radare2/test/../bins/other/projects/test/rc'

[XX] db/cmd/cmd_print prc text color
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc '. ../bins/other/palette.r2
.. ../bins/src/prc_256.py
e scr.color=3
prc 256
' -
-- stdout


-- stderr
Cannot find script '../bins/other/palette.r2'
python: can't open file '../bins/src/prc_256.py': [Errno 2] No such file or directory
Cannot open ttyname(0) (null)

[**]                              db/cmd/cmd_print_misc     2775 OK        86 BR      104 XX       10 FX
[**]                                 db/cmd/cmd_project     2775 OK        86 BR      104 XX       10 FX
[**]                                  db/cmd/cmd_search     2781 OK        86 BR      104 XX       10 FX
[XX] db/cmd/cmd_ph ph file with offsets
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'e io.va=0
ph sha1 $s-32 @ 32
!rahash2 -a sha1 -f 32 -t 4898  ../bins/elf/analysis/x86-helloworld-gcc
ph sha1 $s @0
!rahash2 -a sha1  ../bins/elf/analysis/x86-helloworld-gcc
ph sha1 @s:fofofofo
' bins/elf/analysis/x86-helloworld-gcc
-- stdout
--- .a	2020-04-09 22:55:15.422123000 +0000
+++ .b	2020-04-09 22:55:15.422173000 +0000
@@ -1,5 +1,3 @@
1fa8a7fd5c661af321d09f5951b35684f209f815
-../bins/elf/analysis/x86-helloworld-gcc: 0x00000020-0x00001322 sha1: 1fa8a7fd5c661af321d09f5951b35684f209f815
e218202875386cf4a0cd3ce22490fcd48db91491
-../bins/elf/analysis/x86-helloworld-gcc: 0x00000000-0x00001322 sha1: e218202875386cf4a0cd3ce22490fcd48db91491
b9a2dc76a3571526786cf651570df206a93f63fa


-- stderr
rahash2: Cannot open '../bins/elf/analysis/x86-helloworld-gcc'
rahash2: Cannot open '../bins/elf/analysis/x86-helloworld-gcc'

[**]                                    db/cmd/cmd_pipe     2844 OK        88 BR      105 XX       10 FX
[**]                                     db/cmd/cmd_pix     2844 OK        88 BR      105 XX       10 FX
[**]                                      db/cmd/cmd_pi     2844 OK        88 BR      105 XX       10 FX
[**]                                     db/cmd/cmd_pfd     2844 OK        88 BR      105 XX       10 FX
[**]                                   db/cmd/cmd_print     2844 OK        88 BR      105 XX       10 FX
[**]                                      db/cmd/cmd_ph     2844 OK        88 BR      105 XX       10 FX
[**]                                  db/cmd/cmd_pf_elf     2844 OK        88 BR      105 XX       10 FX
[**]                              db/cmd/cmd_search_hit     2858 OK        88 BR      105 XX       10 FX
[**]                                     db/cmd/cmd_pf2     2864 OK        88 BR      105 XX       10 FX
[XX] db/cmd/cmd_zignature zfs on static glibc
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'aac
zfs ../bins/other/sigs/libc6_2.27-3ubuntu1_amd64.sig~silence
f@F:flirt
' bins/elf/static-glibc-2.27
-- stdout


-- stderr
Can't open ../bins/other/sigs/libc6_2.27-3ubuntu1_amd64.sig

[**]                               db/cmd/cmd_zignature     2934 OK        89 BR      106 XX       10 FX
[**]                                      db/cmd/cmd_pf     2934 OK        89 BR      106 XX       10 FX
[**]                                     db/cmd/cmd_pdx     2934 OK        89 BR      106 XX       10 FX
[**]                                db/cmd/cmd_pd_bytes     2934 OK        89 BR      106 XX       10 FX
[**]                                     db/cmd/cmd_pdc     2934 OK        89 BR      106 XX       10 FX
[**]                                     db/cmd/cmd_pdr     2934 OK        89 BR      106 XX       10 FX
[**]                                 db/cmd/cmd_pd_bugs     2950 OK        91 BR      106 XX       10 FX
[**]                                     db/cmd/cmd_pif     2950 OK        91 BR      106 XX       10 FX
[**]                                     db/cmd/cmd_pd2     3015 OK        91 BR      106 XX       10 FX
[**]                                      db/cmd/cmd_pc     3089 OK        91 BR      106 XX       10 FX
[**]                                     db/cmd/cmd_pae     3089 OK        91 BR      106 XX       10 FX
[XX] db/cmd/cmd_open o+ writable maps fix
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'o+ ../bins/elf/ls
om
# Possible parallelism problem
# ?e
# pv1
# $old=`pv1`
# wx 0x90
# pv1
# wx `$old?`
# pv1
' --
-- stdout


-- stderr
cannot open file ../bins/elf/ls
```

**Comment by radare** [comment]
Maybe i missed some tests but i sed all the FILE=../ oh right there are more places where the ../bins is used

File: binr/r2r/r2r.c [hunk 1]
```diff
@@ -81,7 +81,7 @@ static bool r2r_chdir(const char *argv0) {
 		char *p = strstr (src_path, R_SYS_DIR "binr"R_SYS_DIR"r2r"R_SYS_DIR"r2r");
 		if (p) {
 			*p = 0;
-			strcat (src_path, R_SYS_DIR"test"R_SYS_DIR"new");
+			strcat (src_path, R_SYS_DIR"test"R_SYS_DIR);
```
Comment (thestr4ng3r) [inline binr/r2r/r2r.c:84]:
```suggestion
			strcat (src_path, R_SYS_DIR"test");
```

**Comment by XVilka** [comment]
Can you please rebase and move these new tests?

**Comment by radare** [comment]
yes, i was just afraid of losing tests when moving them

**Comment by radare** [comment]
ive verified no tests has been lost. lets wait for the CI

**Comment by XVilka** [comment]
```
[XX] db/anal/tricore TriCore return_0.elf
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc drp ../bins/tricore/return_0.elf
-- stdout


-- stderr
[r] Cannot open '../bins/tricore/return_0.elf'

-- exit status: 1
[XX] db/anal/java java f_info s
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'java f_info s' zip://../bins/java/example.zip::0
-- stdout
--- .a	2020-04-13 05:20:56.000000000 +0000
+++ .b	2020-04-13 05:20:56.000000000 +0000
@@ -1,7 +0,0 @@
-Field Summary Information:
-  File Offset: 0x00000287
-  Name Index: 40 (a)
-  Descriptor Index: 41 (Z)
-  Access Flags: 0x09 (public static)
-  Field Attributes Count: 0
-  Field Attributes:


-- stderr
ZIP File Error: File could not be opened file name.
[r] Cannot open 'zip://../bins/java/example.zip::0'

-- exit status: 1
[XX] db/anal/java java prototypes a
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'java prototypes a' zip://../bins/java/example.zip::0
-- stdout


-- stderr
ZIP File Error: File could not be opened file name.
[r] Cannot open 'zip://../bins/java/example.zip::0'

-- exit status: 1
[XX] db/anal/java java m_info s
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 's 0x312
java m_info s
' zip://../bins/java/example.zip::0
-- stdout
[XX] db/io/ihex pic18c
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'e asm.arch=pic
e asm.cpu=pic18
pi 2
' ihex://../bins/pic18c/FreeRTOS-pic18c.hex
-- stdout
--- .a	2020-04-13 05:21:27.000000000 +0000
+++ .b	2020-04-13 05:21:27.000000000 +0000
@@ -1,2 +0,0 @@
-goto 0x246e
-return 0


-- stderr
[r] Cannot open 'ihex://../bins/pic18c/FreeRTOS-pic18c.hex'

-- exit status: 1
[XX] db/io/zip io-zip entrypoints for c.class
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc ieq zip://../bins/java/example.zip::0
-- stdout


-- stderr
ZIP File Error: File could not be opened file name.
[r] Cannot open 'zip://../bins/java/example.zip::0'

-- exit status: 1
[XX] db/io/zip io-zip entrypoints for c.class
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc ieq zip://../bins/java/example.zip//research_data/experiment/decompiler/dumped_jar_dec/c.class
-- stdout
--- .a	2020-04-13 05:21:28.000000000 +0000
+++ .b	2020-04-13 05:21:28.000000000 +0000
@@ -1,4 +0,0 @@
-0x000002af
-0x000002da
-0x00000312
```

**Comment by radare** [comment]
i had to commit 2 files in r2-bins that will make all prs red

**Comment by radare** [comment]
its green, can i merge and end this pain?

**Comment by radare** [comment]
it is fully green, just a ray condition cloning bins, i just push -f to rerun the tasks so u can see my point

**Pull request merged**